Mathlib/Algebra/Group/Fin/Basic.lean

```diff
@@ -111,6 +111,12 @@ lemma lt_sub_iff {n : ℕ} {a b : Fin n} : a < a - b ↔ a < b := by
 lemma sub_le_iff {n : ℕ} {a b : Fin n} : a - b ≤ a ↔ b ≤ a := by
   rw [← not_iff_not, Fin.not_le, Fin.not_le, lt_sub_iff]
 
+lemma sub_val_lt_sub {n : ℕ} {i j : Fin n} (hij : i ≤ j) : (j - i).1 < n - i.1 := by
```

Comment:
Are you sure that this is the right file for these two results? They do not seem to have anything to do with group, perhaps they should live in `Data/Fin/Basic`.